js/ui/components/clubhouse.js

```diff
@@ -458,6 +458,10 @@ class ClubhouseNotificationBanner extends MessageTray.NotificationBanner {
     }
 
     _addActions() {
+        // Don't add actions if pagination is not initialized yet
+        if (this._textIdx === undefined || this._textPages === undefined)
+            return;
```

Comment:
The fact that we have to special case this both here and in `this._inLastPage()` made me think that the approach we're using to set the text is actually not the best here.
Instead of saving the text body beforehand, and re-injecting it later, I wonder if we couldn't instead override e.g. the `setBody()` method and assign `this._textPages`, etc, from there. My goal would be to remove checks for `this._textPages` and `this._textIdx` from these two methods.

Reply:
Alternatively, if that's not possible, I'd split out a `this._isPaginationReady()` method or something like that.

Reply:
Done, I've overridden the `setBody` method and add the text split there with a new flag attribute `_paginationReady` to do this only the first time the method is called in the parent constructor, because the `setBody` method is called again on page change and in that case we don't want to split the text again.